setup.py

```diff
@@ -71,7 +71,16 @@
         **extra
     )
 except Exception as e:
-    if isinstance(e.message, str) and 'Could not run curl-config' in e.message:
+    if ((hasattr(e, 'args') and
```

Comment:
i dunno the answer to this question, a 3 min google was not helpful, but maybe there is a proper way of checking to see if a system package is installed and we include that in here somewhere instead of trying to catch the error message